getting-started/quick-start.md

```diff
@@ -65,9 +75,9 @@ Amber works with `postgresql` \(default\), `mysql`, or `sqlite`.
 
 If you don’t already have one of these installed, please follow the guides provided by each database maintainer:
 
-* [Postgresql Installation Guides](https://wiki.postgresql.org/wiki/Detailed_installation_guides)  
-* [MySQL Installation Guides](https://dev.mysql.com/doc/refman/8.0/en/installing.html)  
-* [Tutorial on installing SQLite](https://www.tutorialspoint.com/sqlite/sqlite_installation.htm)  
+- [Postgresql Installation Guides](https://wiki.postgresql.org/wiki/Detailed_installation_guides)
+- [MySQL Installation Guides](https://dev.mysql.com/doc/refman/8.0/en/installing.html)
+- [Tutorial on installing SQLite](https://www.tutorialspoint.com/sqlite/sqlite_installation.htm)
```

Comment:
Why change all of these * to -? That's adding a lot of lines to your pull that are unrelated to the purpose of it.

Reply:
VS Code and Github defaults to CommonMark standard for Markdown syntax.  Is there a particular reason that you're using `*` for unordered lists instead `-`?